documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -0,0 +1,233 @@
++
+For example:
++
+[source,shell,subs="+quotes,attributes"]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  annotations:
+    kubernetes.io/createdby: aws-ebs-dynamic-provisioner
+    pv.kubernetes.io/bound-by-controller: "yes"
+    pv.kubernetes.io/provisioned-by: kubernetes.io/aws-ebs
+  creationTimestamp: "<date>"
+  finalizers:
+  - kubernetes.io/pv-protection
+  labels:
+    failure-domain.beta.kubernetes.io/region: eu-west-1
+    failure-domain.beta.kubernetes.io/zone: eu-west-1c
+  name: pvc-7e226978-3317-11ea-97b0-0aef8816c7ea
+  resourceVersion: "39431"
+  selfLink: /api/v1/persistentvolumes/pvc-7e226978-3317-11ea-97b0-0aef8816c7ea
+  uid: 7efe6b0d-3317-11ea-a650-06e1eadd9a4c
+spec:
+  accessModes:
+  - ReadWriteOnce
+  awsElasticBlockStore:
+    fsType: xfs
+    volumeID: aws://eu-west-1c/vol-09db3141656d1c258
+  capacity:
+    storage: 100Gi
+  *claimRef:*
+    *apiVersion: v1*
+    *kind: PersistentVolumeClaim*
+    *name: data-0-my-cluster-kafka-2*
+    *namespace: myproject*
+    *resourceVersion: "39113"*
+    *uid: 54be1c60-3319-11ea-97b0-0aef8816c7ea*
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: failure-domain.beta.kubernetes.io/zone
+          operator: In
+          values:
+          - eu-west-1c
+        - key: failure-domain.beta.kubernetes.io/region
+          operator: In
+          values:
+          - eu-west-1
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: gp2-retain
+  volumeMode: Filesystem
+----
++
+In the example, the following properties are deleted:
++
+[source,shell,subs="+quotes,attributes"]
+----
+claimRef:
+  apiVersion: v1
+  kind: PersistentVolumeClaim
+  name: data-0-my-cluster-kafka-2
+  namespace: myproject
+  resourceVersion: "39113"
+  uid: 54be1c60-3319-11ea-97b0-0aef8816c7ea
+----
+
+. Deploy the Cluster Operator.
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl apply -f install/cluster-operator -n _my-project_
+----
+
+. Recreate your cluster.
++
+Follow the steps depending on whether or not you have all the `KafkaTopic` resources needed to recreate your cluster.
++
+--
+*_Option 1_*: If you have *all* the `KafkaTopic` resources that existed before you lost your cluster, including internal topics such as committed offsets from `__consumer_offsets`:
+
+. Recreate all `KafkaTopic` resources.
++
+It is essential that you recreate the resources before deploying the cluster, or the Topic Operator will delete the topics.
+
+. Deploy the Kafka cluster.
++
+For example:
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl apply -f _kafka.yaml_
+----
+--
++
+--
+*_Option 2_*: If you do not have all the `KafkaTopic` resources that existed before you lost your cluster:
+
+. Deploy the Kafka cluster, as with the first option, but without the Topic Operator by removing `topicOperator: {}` from the Kafka resource before deploying.
```

Comment:
I would say `topicOperator` property (or field) without the curly braces because they mean default configuration while the user could have a `Kafka` resource with the topic operator configuration different from the default. In this way the user will be not confused not having just curly braces which is mentioned in the doc.

Reply:
Right. I get it.